frontend/src/components/layout/navigation/UpgradeButton.tsx

```diff
@@ -0,0 +1,25 @@
+import { useLocalization } from "@fluent/react";
+import styles from "./UpgradeButton.module.scss";
+import { useGaViewPing } from "../../../hooks/gaViewPing";
+import Link from "next/link";
+import { trackPurchaseStart } from "../../../functions/trackPurchase";
+
+export const UpgradeButton = (): JSX.Element => {
+  const { l10n } = useLocalization();
+  const upgradeButtonRef = useGaViewPing({
+    category: "Upgrade",
```

Comment:
I think we need this category here to get it to flow into the correct reports in Google Analytics:

```suggestion
    category: "Purchase Button",
```